src/cljs/main/broadfcui/common/table.cljs

```diff
@@ -394,17 +395,16 @@
    :get-default-props get-default-table-props
    :get-initial-state get-initial-table-state
    :render render-table
-   :execute-search (fn [{:keys [this]}] (react/call :refresh-rows this))
```

Comment:
Since execute-search is just calling refresh-rows, I deleted it and made callers call refresh-rows directly